examples/http_server.md

```diff
@@ -59,23 +59,22 @@ Then navigate to `http://localhost:8080/` in a browser.
 
 ### Using the `std/http` library
 
-> ℹ️ Since stabilization of _native_ HTTP server in 1.13, usage of `std/http` is
-> discouraged. The module is planned to be deprecated in the future releases.
-
```

Comment:
Could you please add a note here, saying that `std/http` now uses native bindings as well? Also I think we should keep note of "legacy" server as well until it's removed from std altogether.

Reply:
Added some additional notes, let me know what think!